provision/roles/node/tasks/main.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: install git
+  become: yes
+  apt: name=git state=latest
+
+- name: download new version of nodejs
+  become: yes
+  get_url:
+    url: http://deb.nodesource.com/setup_6.x
+    dest: /tmp/node_setup_6.x
+    mode: 0755
+
+- name: add node.js repository
+  become: yes
+  shell: /tmp/node_setup_6.x
+
+- name: install new version of nodejs
+  become: yes
+  apt:
+    name: nodejs
+    state: latest
+
+- name: install gulp and bower
+  become: yes
+  npm: name={{ item }} global=yes
+  with_items:
+    - gulp
+    - bower
+
+- name: install npm dependencies
+  npm: path={{ project_path }}
```

Comment:
Due to bug with npm module use shell instead. It doesn't install dependencies when `node_modules` directory exists.